src/lib/ll_schedule.c

```diff
@@ -63,7 +63,7 @@
  */
 
 struct ll_schedule_data {
-	struct list_item work[LL_PRIORITIES];	/* list of ll per priority */
+	struct list_item tasks;			/* list of ll tasks */
```

Comment:
What's the rational behind this ? Why would we want to remove a feature ?

Reply:
It's not being removed, rather optimised. We don't need list per priority anymore, because now tasks are immediately inserted into one list by their priority (sorted).

Reply:
Yes, and if we wanted increase the number of priorities (e.g. to 99 as we talked in #1089), we would not have to allocate queues per all priorities.